content/configuration/_index.md

```diff
@@ -51,3 +51,11 @@ Or:
 ```
 oc exec -n openshift-sdn $(oc get pods -n openshift-sdn -l app=sdn --no-headers=true -o custom-columns=:metadata.name|head -n1) cat /config/{kube-proxy-config,sdn-config}.yaml
 ```
+
+# Making Master Unscheduleable/Scheduleable
+To configure master as unscheduleable (when UPI installation without setting this prior to the install):
+```
```

Comment:
Can you add a new line between the content and the code? Thanks!

Reply:
fixed this